src/Infrastructure/Identity/CurrentUserService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Codidact.Application.Common.Interfaces;
+using Codidact.Infrastructure.Identity;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.Security.Claims;
+
+namespace Codidact.Infrastructure.Identity
+{
+    public class CurrentUserService : ICurrentUserService
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        public CurrentUserService(IHttpContextAccessor httpContextAccessor)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        /// <summary>
+        /// Retreives the user id of the current user.
+        /// </summary>
+        public string GetUserId()
+        {
+            return _httpContextAccessor.HttpContext?.User?.FindFirstValue(ClaimTypes.NameIdentifier);
+
```

Comment:
Extra blank line

Reply:
Fixed in https://github.com/codidact/core/pull/29/commits/55abf50aba5742c2e43c128da7af67e2c8b2d41e